src/Foundation/Defaults.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Comparables.swift
+//
+//  Created by Logan Wright on 10/25/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+import Foundation
+
+protocol SettingsKeyAccessible {
+    var key: String { get }
+    var defaults: NSUserDefaults { get }
+}
+
+extension SettingsKeyAccessible {
+    var defaults: NSUserDefaults {
```

Comment:
I think you could test this by having either a mock protocol, or you could define an implementation for each target that overrides this in a different way.

Adding injection would add more requirements for implementation I think.